src/models/outbound/pispOTPValidate.interface.ts

```diff
@@ -21,22 +21,40 @@
  * Gates Foundation
  - Name Surname <name.surname@gatesfoundation.com>
 
- - Kevin Leyow <kevin.leyow@modusbox.com>
+ - Kevin Leyow - kevin.leyow@modusbox.com
  --------------
  ******/
+import {
+  ControlledStateMachine,
+  PersistentModelConfig, StateData
+} from '~/models/persistent.model'
+import { Method } from 'javascript-state-machine'
+import { ThirdpartyRequests } from '@mojaloop/sdk-standard-components';
+import {
+  thirdparty as tpAPI
+} from '@mojaloop/api-snippets'
+import { PubSub } from '~/shared/pub-sub'
+export interface PISPOTPValidateStateMachine extends ControlledStateMachine {
+  validateOTP: Method
+  onValidateOTP: Method
+}
+
+export interface PISPOTPValidateModelConfig extends PersistentModelConfig {
+  pubSub: PubSub
+  thirdpartyRequests: ThirdpartyRequests
+  requestProcessingTimeoutSeconds: number
+}
+
+export interface PISPOTPValidateData extends StateData {
```

Comment:
I think you can import `ConsentRequestsValidateState` from `src/interface/outbound/api_interfaces/index.ts` 

and use it as a generic parameter when extending StateData

like here: https://github.com/mojaloop/thirdparty-scheme-adapter/blob/master/src/models/pispTransaction.interface.ts#L86
so you don't have to redefine currentState

```suggestion
export interface PISPOTPValidateData extends StateData<ConsentRequestsValidateState> {
```

Reply:
Ah I didn't see that you already checked that in with a PR. Thought that it was something you were going to do. done.